app/dashboard/views.py

```diff
@@ -1776,7 +1776,10 @@ def helper_handle_approvals(request, bounty):
 
                 maybe_market_to_github(bounty, 'work_started', profile_pairs=bounty.profile_pairs)
                 maybe_market_to_slack(bounty, 'worker_approved')
-                maybe_market_to_user_slack(bounty, 'worker_approved')
+                try:
```

Comment:
longer term it may be a good idea to move these jobs to celery instead of pageload